licenses/management/commands/publish.py

```diff
@@ -0,0 +1,148 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+
+from django.core.management import BaseCommand
+
+from cc_licenses.settings.base import TRANSLATION_REPOSITORY_DIRECTORY
+
+# Go to cc-licenses-data and do something
+GO_TO_TRANSLATIONS_REPO = f"cd {TRANSLATION_REPOSITORY_DIRECTORY} && "
+PULL_BRANCHES_DOWN_FROM_DEVELOP = "git checkout develop && git pull"
+PULL_DOWN_TRANSLATION_BRANCHES = (
+    GO_TO_TRANSLATIONS_REPO + PULL_BRANCHES_DOWN_FROM_DEVELOP
+)
+
+
+def run_django_distill():
+    """Outputs the build directory
+    """
+    my_env = os.environ.copy()
+    my_env["DJANGO_SETTINGS_MODULE"] = "cc_licenses.settings.publish"
+    cmd = "python manage.py distill-local --quiet --force"
+    return subprocess.run(
+        cmd, shell=True, check=True, text=True, input="YES", env=my_env,
+    )
```

Comment:
It's a good idea to avoid `shell=True` when we can. It tends to break when varying parts of the command have spaces, quotes, etc, and can even open up security holes.

In this particular case, it's less of a problem because the command is fixed, but I'd still prefer we consistently not use `shell=True` even in cases where it doesn't matter. That saves future programmers having to look to see if shell=True is really safe on a given call. I just always pass the command as an array of strings, e.g. `["python", "manage.py", "distill-local", "--quiet", "--force"]`. It's a bit tedious, but safe.

For other cases in this PR, you can pass `cwd=` to change directories before running the command. `check=True` will raise an exception if the command fails, if you don't want to write code to explicitly check the return status, so you can just run commands one after another and don't need ` && `.

Reply:
thanks. Making changes now. The above is very helpful.